sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -10,12 +10,15 @@
 import pytest
 
 from azure.core import MatchConditions
-from azure.core.exceptions import HttpResponseError, ResourceExistsError, ResourceNotFoundError, \
-    ResourceModifiedError, ServiceRequestError, AzureError
-from azure.storage.filedatalake import ContentSettings, DirectorySasPermissions, DataLakeDirectoryClient, \
-    generate_file_system_sas, FileSystemSasPermissions
+from azure.core.exceptions import (HttpResponseError, ResourceExistsError, ResourceNotFoundError,
+                                   ResourceModifiedError, ServiceRequestError, AzureError)
```

Comment:
nit: Imports still a bit wonky in these test files :)
For example, this one should be:

```python
from azure.core.exceptions import (
    HttpResponseError,
    ResourceExistsError,
    ResourceNotFoundError,
    ResourceModifiedError,
    ServiceRequestError,
    AzureError
)
```

And technically, if you're touching it, should probably sort it too :). But I'm also fine if you just revert this stuff and we address it when we migrate to test proxy.

Reply:
*facepalm* I was only looking at FSC & DSC on my last pass-through. Will address!